### ISSUE TITLE
Font Library: No message appears when pressing the Update button

### ISSUE THREAD

**Issue by t-hamano** [WordPress/gutenberg#54601]
## What problem does this address?

In the font library, for example, when you change the variation you use, you can click the Update button. When you click that button, the state is indeed saved, but no message is displayed.

## What is your proposed solution?

In my opinion, there is a solution to either of the following:

- Use the Snackbar component to notify that settings have been properly updated.
- Remove Update button.

Personally, I prefer the latter approach. As I understand it, the behavior of the Update button has the same role as the Save button in the Global Styles. Also, the font variation setting is part of the global style.

I think whether or not to save changes made in global styles should be centralized in one button.

## Screencast

https://github.com/WordPress/gutenberg/assets/54422211/2d9c12cc-c382-4f0e-8343-9a009e510289

**Comment by madhusudhand** [comment]
@jasmussen Can you share your ideas on the messaging UI?

**Comment by jasmussen** [comment]
Yes, I touched on that in [this PR review](https://github.com/WordPress/gutenberg/pull/54566#issuecomment-1725042244). I'm suggesting there do _either of the following_. Either when you press Update, the dialog is immediately closed, and the multi entity saving interface is opened, this one:

<img width="1615" alt="multi entity saving" src="https://github.com/WordPress/gutenberg/assets/1204802/1fa90178-bed8-4cc9-8ec8-08ee79c60cb2">

**Or**, when the Update button is pressed, you are taken to your _library_ tab in the dialog. After all, you just enqueued it to be in your library.

<img width="1615" alt="library" src="https://github.com/WordPress/gutenberg/assets/1204802/e5adbbfe-8c2e-40c1-a56a-a070b9795981">

I think we should avoid snackbars in this iteration, the basic flow should be in place before we add notices to explain.

**Comment by afercia** [comment]
Raising this issue to bug and adding the Accessibility label, as the lack of visible and audible feedback isn't ideal.

**Comment by akasunil** [comment]
A simple notice could work, in my opinion. We also use notice for font installation. Notice text can be something like `Font settings were updated successfully`

<img width="920" alt="image" src="https://github.com/user-attachments/assets/5d75beec-0ced-4cd2-9610-2acc174c2de0">

**Comment by t-hamano** [comment]
To ideally solve this problem, I think the first step would be to manage the notification as a state instead of context. The notification is only active on that screen and is supposed to be unmounted when the screen changes.

Currently, [The notification is managed as a context](https://github.com/WordPress/gutenberg/blob/313246a01f18e504dabd8313e7eacca728332bcd/packages/edit-site/src/components/global-styles/font-library-modal/context.js#L57), and notification status is preserved even when switching screens, so there is redundant processing to reset the notification status ([example](https://github.com/WordPress/gutenberg/blob/313246a01f18e504dabd8313e7eacca728332bcd/packages/edit-site/src/components/global-styles/font-library-modal/index.js#L62-L65)).

I will also address this issue as part of #58428.
